Obsidian/Server.cs

```diff
@@ -290,6 +289,45 @@ public async Task StartServerAsync()
             this.Logger.LogWarning("Server is shutting down...");
         }
 
+        internal async Task ExecuteCommand(string input)
+        {
+            var console = new ConsoleClient(Logger, this);
+            var context = new CommandContext(Commands._prefix + input, console, this, CommandIssuer.Console);
+            try
+            {
+                await Commands.ProcessCommand(context);
+            }
+            catch (CommandArgumentParsingException)
+            {
+                Logger.LogError($"Invalid arguments! Parsing failed.");
+            }
+            catch (CommandExecutionCheckException)
+            {
+                Logger.LogError($"You can not execute this command.");
+            }
+            catch (CommandNotFoundException)
+            {
+                Logger.LogError($"No such command was found.");
+            }
+            catch (NoSuchParserException)
+            {
+                Logger.LogError($"The command you executed has a argument that has no matching parser.");
+            }
+            catch (InvalidCommandOverloadException)
+            {
+                Logger.LogError($"No such overload is available for this command.");
+            }
```

Comment:
I looked through `CommandHandler.ProcessComand` and whenever an exception is thrown, it always seems to set the `Exception.Message`, why aren't you using that?

Reply:
I copied this code from [here](https://github.com/ObsidianMC/Obsidian/blob/eab18e7946db785c49302046cfdbfed68d46a077/Obsidian/Server.cs#L303)